src/app/favoritePlaces/googlemap.html

```diff
@@ -0,0 +1,11 @@
+<div class="map-wrapper">
```

Comment:
I will create a new folder for directive, and add it to components. I think that's a better way, as each directive should have its own separate template HTML file. What do u think?